enterprise/models.py

```diff
@@ -2368,6 +2371,13 @@ class EnterpriseCustomerReportingConfiguration(TimeStampedModel):
         verbose_name=_("Delivery Method"),
         help_text=_("The method in which the data should be sent.")
     )
+    enable_compression = models.BooleanField(
+        default=True,
+        help_text=_(
+            "Specifies whether report should be compressed. Without compression files will not be password protected "
+            "or encrypted."
```

Comment:
I know compression is needed to set a password but how is encryption dependent on compression?

Reply:
shouldn't this change in model have a migration? I can't seem to find any migration file in the PR